cpp-low-level/Dockerfile

```diff
@@ -17,13 +17,17 @@ FROM --platform=linux/riscv64 riscv64/ubuntu:22.04
 LABEL io.sunodo.sdk_version=0.1.0
 LABEL io.cartesi.rollups.ram_size=128Mi
 
+ENV MACHINE_EMULATOR_TOOLS_VERSION=0.12.0
```

Comment:
ENV or ARG?

Reply:
Yes, ARG would be better to change during build.